src/backend/distributed/executor/adaptive_executor.c

```diff
@@ -242,10 +242,10 @@ typedef struct DistributedExecution
 	bool raiseInterrupts;
 
 	/*
-	 * Flag to indicate whether the query is running in a distributed
-	 * transaction.
+	 * The transaction blocks usage indicates whether the current execution
```

Comment:
/*
	 * useRemoteTransactionBlocks is used to manage remote transactions. It
	 * could state one of the followings:
	 * - opening a remote transaction is required
	 * - opening a remote transaction is disallowed
	 * - opening a remote transaction does not matter, so it is allowed but not required.
	 */

my comment was like this, you can use it if you find it more descriptive.